app/models/referee_interface/reference_feedback_form.rb

```diff
@@ -16,7 +16,9 @@ def save
       ReceiveReference.new(
         reference: reference,
         feedback: feedback,
-      ).save
+      ).save!
+
+      true
```

Comment:
How does the presence validation for `feedback` work with this change?

Reply:
If the feedback isn't provided and form isn't `valid?` it'll return nil early:

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/686d8f8c6e5141a220cf5fc33108bfe7140d6d6d/app/models/referee_interface/reference_feedback_form.rb#L14

Reply:
should have spotted that!